datalad/core/local/create.py

```diff
@@ -516,3 +457,115 @@ def __call__(
 
         res.update({'status': 'ok'})
         yield res
+
+
+def _setup_git_repo(path, add_to_git, initopts=None, fake_dates=False):
+    """Create and configure a repository at `path`
+
+    Parameters
+    ----------
+    path: str or Path
+      Path of the repository
+    add_to_git: dict
+      To add records for any repo component that needs to be passed
+      to git-add as a result of the setup procedure.
+    initopts: list, optional
+      Git options to be passed to the GitRepo constructor
+    fake_dates: bool, optional
+      Passed to the GitRepo constructor
+
+    Returns
+    -------
+    GitRepo
+    """
+    lgr.info("Creating a new git repo at %s", path)
+    tbrepo = GitRepo(
+        path,
+        create=True,
+        create_sanity_checks=False,
+        git_opts=initopts,
+        fake_dates=fake_dates)
+    # place a .noannex file to indicate annex to leave this repo alone
+    stamp_path = Path(tbrepo.path) / '.noannex'
+    stamp_path.touch()
+    add_to_git[stamp_path] = {
+        'type': 'file',
+        'state': 'untracked'}
+    return tbrepo
+
+
+def _setup_annex_repo(path, add_to_git, initopts=None, fake_dates=False,
+                      description=None):
+    """Create and configure a repository at `path`
+
+    This includes a default setup which files to never annex.
```

Comment:
s/which/of &/ ?

Or perhaps "This includes a default setup of `annex.largefiles`."

Reply:
Yes, true, done (the full replacement).